google-cloud-tools-plugin/google-account/src/com/google/cloud/tools/intellij/analytics/UsageTrackerConfigurableProvider.java

```diff
@@ -42,11 +41,7 @@ public boolean canCreateConfigurable() {
     // For now we can hide Google entirely if usage tracking isn't available as there are no
     // other Google related account settings in the IJ UI.
     // Create a sub-menu item for the cloud SDK and hide the usage tracker if not avaible
-    if (PlatformUtils.isIntelliJ() && UsageTrackerManager.getInstance()
-        .isUsageTrackingAvailable()) {
-      return true;
-    } else {
-      return false;
-    }
+    return PlatformUtils.isIntelliJ() && UsageTrackerManager.getInstance()
```

Comment:
nit: Ditto about formatting. Since we're touching all of these files it might be a good time to ensure the google-java-formatter is up-to-date and correctly running on these files.

Reply:
Should be.  I'm running the gradle rule.  Although it doesn't appear to be doing much so maybe it is misconfigured.